.github/workflows/deploy.yml

```
@@ -0,0 +1,78 @@
+          host: ${{ secrets.DEV_HOST }}
+          username: ${{ secrets.DEV_USERNAME }}
+          password: ${{ secrets.DEV_PASSWORD }}
+          port: ${{ secrets.DEV_PORT }}
+          script: |
+            set -euo pipefail
+            echo "${{ secrets.NCP_SECRET_KEY }}" | docker login ${{ secrets.NCP_CONTAINER_REGISTRY }} -u "${{ secrets.NCP_ACCESS_KEY }}" --password-stdin
+            docker pull ${{ secrets.NCP_CONTAINER_REGISTRY }}/express2:${{ github.sha }}
+            docker stop app || true
+            docker rm app || true
+            docker run -d -p 8080:8080 --name app ${{ secrets.NCP_CONTAINER_REGISTRY }}/express2:${{ github.sha }}
+            docker image prune -f
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)